apps/src/StudioApp.js

```diff
@@ -1793,7 +1794,8 @@ StudioApp.prototype.configureDom = function (config) {
   // TODO (cpirich): make conditional for applab
   var belowViz = document.getElementById('belowVisualization');
   var referenceArea = document.getElementById('reference_area');
-  if (referenceArea) {
+  //TODO : this will get less ugly after we remove the flag - epeach
+  if ((config.app && (!config.app.includes('lab') || (!experiments.isEnabled('resourcesTab') && config.app.includes('lab'))) || !config.app) && referenceArea) {
```

Comment:
This is very difficult to parse, and although it will be somewhat less ugly without the flag, it's still not great.

In general, I don't think we want to just look to see if the app has `lab` in the name, as in the future we could add some new app that was not expecting this behavior (for example if we renamed studio to playlab).

In TopInstructions.jsx, we use `state.instructions.noInstructionsWhenCollapsed` to determine whether we should be showing CSP top instructions vs. CSF. Does it make sense to use that same check here? If so, I would then also add a comment explaining why that's the check we use, as it's not entirely intuitive from the name.